tests/examples_e2e/setup_module/main.tf

```diff
@@ -15,8 +15,9 @@
 locals {
   prefix = "${var.prefix}-${var.timestamp}${var.suffix}"
   jit_services = [
-    "storage.googleapis.com",  # no permissions granted by default
-    "sqladmin.googleapis.com", # roles/cloudsql.serviceAgent
+    "storage.googleapis.com",       # no permissions granted by default
+    "sqladmin.googleapis.com",      # roles/cloudsql.serviceAgent
+    "secretmanager.googleapis.com", # roles/cloudkms.cryptoKeyEncrypterDecrypter
```

Comment:
According to [IAM docs](https://cloud.google.com/iam/docs/service-agents), this service account doesn't have any role granted by default, if true - then change comment to `# no permissions granted by default`